plugins/inputs/docker/docker.go

```diff
@@ -82,6 +85,9 @@ var sampleConfig = `
   ##   To use environment variables (ie, docker-machine), set endpoint = "ENV"
   endpoint = "unix:///var/run/docker.sock"
 
+  ## Set to true to collect Swarm metrics(desired_replicas, running_replicas)
+  swarm_enabled = false
```

Comment:
Maybe this should be named `gather_services = true` or `gather = ["services"]` since the similar docker command is simply `docker service`.